.eslintrc.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
For future consideration: If Prettier's already in the project, may be good to enforce via [`eslint-plugin-prettier`](https://github.com/prettier/eslint-plugin-prettier).

Reply:
I intended to add it, fixed in ed35e669b37c35e8f34fc11bd7ee5622bbc75c40